tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -529,6 +530,14 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
   pm.addNestedPass<FuncOp>(createCanonicalizerPass());
   pm.addNestedPass<FuncOp>(createCSEPass());
   pm.addNestedPass<FuncOp>(createCanonicalizerPass());
+
+  bool enable_op_schedule = false;
+  tensorflow::ReadBoolFromEnvVar("DISC_ENABLE_OP_SCHEDULE", enable_op_schedule,
```

Comment:
Does this pass cause any native optimization? If not, let's enable it by default, we have too many flags...

Reply:
Ok, we havent found a case which schedule pass will cause negative effect.

Reply:
Fixed.